src/components/Button/index.stories.js

```diff
@@ -24,14 +24,14 @@ stories.add('All states', () => (
 
         <div>
           <Heading level={3}>States</Heading>
-          <Button intent={intent} style={{ marginRight: 'var(--f-space--m)' }}>
+          <Button intent={intent} margin={'0 var(--f-space--m) 0 0'}>
             {intent}
           </Button>
           <Button
             intent={intent}
             className={`is-${intent}`}
             isLoading
-            style={{ marginRight: 'var(--f-space--m)' }}
+            margin={'0 var(--f-space--m) 0 0'}
```

Comment:
Is this a good practice?
I wondered if for this kind of stuff using `css` wouldn't be better, wdyt?

Reply:
I was wondering the same thing, the reason I went for this is that `css` will "extend" and add extra css, so there will be 2 `margin`, plus you need to use the styled-components babel plugin for `css` to work.

It's not a big deal, either way, just thought this would be slightly easier 🤷‍♂️

Reply:
👌